### PR TITLE
Add the output of helmfile to the action outputs

### PR DESCRIPTION
**Pull request by metiletan**
It gives the possibility to use helmfile output in other CI steps.

### CODE REVIEW

**Review by romcheg — COMMENTED** [review]
I would personally call these two as `helmfileStdout` and `helmfileStderr` respectively.

File: src/main.ts [hunk 1]
```diff
@@ -39,7 +39,21 @@ async function run(): Promise<void> {
       options
     );
 
+    let helmfileOutput = '';
+    let helmfileError = '';
```
Comment (romcheg) [inline src/main.ts:43]: I would personally call these two as `helmfileStdout` and `helmfileStderr` respectively.

**Review by romcheg — APPROVED** [review]
Looks good to me. Here's my non-binding approve :)

**Comment by yxxhero** [comment]
@metiletan  please fix ci issue.

**Comment by metiletan** [comment]
> @metiletan please fix ci issue.

done

**Comment by yxxhero** [comment]
@metiletan BTW. please update the readme.md. thanks so much.

**Pull request merged**